cpp/arcticdb/async/async_store.hpp

```diff
@@ -22,227 +22,263 @@ namespace arcticdb::async {
 template<class ClockType = util::SysClock>
 class AsyncStore : public Store {
 public:
-    AsyncStore(
-        std::shared_ptr<storage::Library> library,
-        const arcticdb::proto::encoding::VariantCodec &codec) :
-        library_(std::move(library)),
-        codec_(new arcticdb::proto::encoding::VariantCodec{codec}) {
+    AsyncStore(std::shared_ptr<storage::Library> library, const arcticdb::proto::encoding::VariantCodec& codec)
+        : library_(std::move(library)),
+          codec_(new arcticdb::proto::encoding::VariantCodec{codec})
+    {
```

Comment:
Egyptian brackets please.

Reply:
There are basically 3 things we can do:
```
void function(int param1,
    int param2,
    int param3,
    int param4) {
    //function body
}
```
Pros: most of the function will stay the same
Cons: When wrapping happens the last param and the function body are indented the same and it's kind of hard to tell where params end and where the body starts
```
void function(int param1,
    int param2,
    int param3,
    int param4)
{
    //function body
}
```
Pros: Clear separation between the end of the function params and the start of the body
Cons: Changes each function
```
void function(int param1,
                      int param2,
                      int param3,
                      int param4) {
    //function body
}
```
Pros: Provides some separation (not always) between the body and the end of the params.
Cons: (Personal opinion) looks weird.  Will change lots of functions